tests/integration/test_refreshable_mat_view/test.py

```diff
@@ -249,7 +247,6 @@ def test_simple_append(
 
 
 @pytest.mark.parametrize("with_append", [True, False])
-@pytest.mark.parametrize("if_not_exists", [True, False])
```

Comment:
(Having 128 minor variations of an expensive test is wasteful. This reduces it to 64, but would be nice to improve this further.)